Runtime/Scripts/PeriodicSnapshots.cs

```diff
@@ -212,8 +221,8 @@ public Texture2D CreateTimestampTexture(byte[] bytes, DateTime time, int width,
             textObject.transform.SetParent(transform);
             TextMesh textComponent = textObject.AddComponent<TextMesh>();
             textComponent.font = font;
-            textComponent.text = time.ToString("yyyy-MM-dd HH:mm:ss");
-            textComponent.fontSize = 200;
+            textComponent.text = time.ToString("yyyy-MM-dd HH:mm");
+            textComponent.fontSize = 150;
             textComponent.color = Color.white;
```

Comment:
Moet dit niet instelbaar zijn of een serializedfield omdat de texture ook wit zou kunnen zijn?